contrib/endpoints/src/api_manager/request_handler.cc

```diff
@@ -29,6 +29,17 @@ using google::devtools::cloudtrace::v1::Traces;
 namespace google {
 namespace api_manager {
 
+RequestHandler::RequestHandler(ApiManagerImpl* api_manager,
+                               std::shared_ptr<CheckWorkflow> check_workflow,
+                               std::unique_ptr<Request> request_data)
+    : api_manager_(api_manager),
+      check_workflow_(check_workflow),
+      request_data_(std::move(request_data)) {
+  if (api_manager->IsConfigLoadingSucceeded()) {
```

Comment:
If ApiManager was initialized in the constructor with service_config, then 
 request_handler->GetServiceConfigId() should return config_id.

This affects transcoder_factory.
https://github.com/cloudendpoints/esp/blob/master/src/nginx/module.cc#L75